PriorityQueue/include/PriorityQueue.h

```diff
@@ -1,19 +1,43 @@
 #pragma once
-
-#include <iostream>
 #include <list>
-#include <exception>
+#include <stdexcept>
+
+using empty_queue_except = std::out_of_range;
 
+template <typename T>
+using priorityQueue = std::list<T>;
 
-// any templates?
+template <typename T>
+struct MyComperator {
+	T operator()(T a, T b) {
+		return a - b;
+	}
+};
+
+template <typename T>
 class PriorityQueue {
-   
+
+	MyComperator<T> m_comp;
```

Comment:
You're correct that the default access to a class is private, but in this case it's good practice to be explicit and mark the class members under `private:` section